contrib/ci/dependencies.txt

```diff
@@ -34,6 +34,7 @@ gnutls-devel,			,				gnutls-dev,			gnutls-dev,			libgnutls28-dev:s390x,
 gnutls-utils,			,				,				,				,
 gtk-doc,			gtk-doc,			gtk-doc-tools,			gtk-doc-tools,			,
 jq,				,				,				,				,
+json-glib-devel,		,				libjson-glib-dev,		libjson-glib-dev,		libjson-glib-dev:s390x,
```

Comment:
Since debian/control build-depends still isn't automatically generated (although i have a goal to eventually fix that), make sure you update that too.

Reply:
Done.